trace_viewer/extras/importer/linux_perf/i915_parser.html

```diff
@@ -1,8 +1,8 @@
 <!DOCTYPE html>
 <!--
-Copyright (c) 2012 The Chromium Authors. All rights reserved.
-Use of this source code is governed by a BSD-style license that can be
-found in the LICENSE file.
+ Copyright (c) 2015 The Chromium Authors. All rights reserved.
+ Use of this source code is governed by a BSD-style license that can be
+ found in the LICENSE file.
```

Comment:
These changes shouldn't be needed. If you update @natduca  fixed the presubmit check.